docs/.gitignore

```
@@ -0,0 +1,2 @@
+build/
```

Maybe change to
```suggestion
/build/
```
to avoid that other directories named build are excluded. Possibly also just add it to the main .gitignore?